src/OpenTelemetry.Exporter.Prometheus/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.3.0-alpha.1
```

Comment:
I'm kind of battling with myself.

Do we think we can/should do the same for Instrumentation Libraries? (e.g. OpenTelemetry.Instrumentation.Http)

Also it seems a bit weird to have the following release sequence:

* 1.2.0-rc1
* 1.2.0-rc2
* 1.2.0-rc3
* 1.2.0-rc4
* 1.2.0-rc5
* BOOM there is no 1.2.0 Stable because we're waiting for the spec
* 1.3.0-alpha.1

Reply:
I think what we need to decide in the long run is what our versioning scheme should communicate to end users. Looks like you started this discussion with #917. I'll add some thoughts on that issue to continue the conversation.

Even though it's odd, I think it's ok for now that Prometheus will jump past a 1.2.0 stable release.

>  if OpenTelemetry.Instrumentation.Http is Stable today, do we release it as 1.0.0 or 1.2.0? (I feel it should be 1.2.0)

I also agree that it should be 1.x.0 where x = current core stable version at the time instrumentation goes stable. Again, while odd, I think it'll be ok to be odd up to the point where we've gotten a stable release out of all components.